services/QuillLMS/client/app/bundles/PremiumHub/containers/diagnosticGrowthReports/index.tsx

```diff
@@ -58,35 +71,44 @@ export const DiagnosticGrowthReportsContainer = ({
     passedData: null
   }
 
-  function renderContent() {
+  function handleTabChange(e) {
+    setActiveTab(e.currentTarget.value)
+  }
+
+  function handleSetNoDiagnosticDataAvailable(value: boolean) {
+    setNoDiagnosticDataAvailable(value)
+  }
+
+  function renderButtons() {
+    return(
+      <div className="tabs-for-pages-container">
+        {reportButtons.map(button => {
+          const { tab, displayName, activeIconSrc, inactiveIconSrc } = button
+          return (
+            <button className={`interactive-wrapper performance-type-button overview ${tab === OVERVIEW ? 'active' : ''}`} onClick={handleTabChange} value={tab}>
+              <img alt="" src={tab === OVERVIEW ? activeIconSrc : inactiveIconSrc} />
```

Comment:
should these two have `activeTab === tab` as the condition instead of `tab === OVERVIEW`?